e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -97,7 +98,7 @@ Scenario('HMCTS admin enters hearing details and submits', async (I, caseViewPag
   await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[0]);
   await I.addAnotherElementToCollection();
   await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[1]);
-  await I.completeEvent('Save and continue', { summary: 'summary', description: 'description' });
+  await I.completeEvent('Save and continue', {summary: 'summary', description: 'description'});
```

Comment:
hmm, intentional change?

Reply:
no - must be when I reformatted. should I change things like this back?